lib/OpenQA/WebAPI/Controller/API/V1/Comment.pm

```diff
@@ -82,7 +82,8 @@ text, date of update and the user name that created the comment.
 sub list ($self) {
     my $comments = $self->_comments();
     return unless $comments;
-    $self->render(json => [map { $_->extended_hash } $comments->all]);
+    my $render_markdown = $self->param('render_markdown') // 0;
+    $self->render(json => [map { $_->extended_hash($render_markdown) } $comments->all]);
```

Comment:
Please add explanation for the parameter in the POD string above starting in line 74. That string is rendered into API help, e.g. see https://openqa.opensuse.org/not_found